app/Http/Requests/NavaidRequest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Validation\Rule;
+use Illuminate\Foundation\Http\FormRequest;
+
+class NavaidRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
Do we always want this to be true?

Reply:
The auth should be handled by the middleware and OAuth scopes so I'd say so

Reply:
Sweet